### PR TITLE
Fix/#3 due on

### PR DESCRIPTION
**Pull request by seunggabi**
| 🚥 Fix RM-XXX |
| :-- |
#3 

## 🧰 Changes

Describe in detail what this PR is for.

## 🧬 QA & Testing

Provide as much information as you can on how to test what you've done.

### CODE REVIEW

**Comment by erunion** [comment]
What's the story behind this fix?

**Comment by seunggabi** [comment]
@erunion 
auto created milestone's due_on `00:00:00`
we suppose milestone due_on `23:59:59`

milestone name is `weekly (~ 2022-09-18)`

we will close `2022-09-18 23:59:59`

**Comment by seunggabi** [comment]
https://github.com/orgs/community/discussions/33427

**Comment by seunggabi** [comment]
스크린샷 2022-09-19 07 31 00

File: .github/workflows/auto-create-milestone.yaml [hunk 1]
```diff
@@ -8,14 +8,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: run
-        uses: ./
+      - uses: ./
+        id: run
```
Comment (erunion) [inline .github/workflows/auto-create-milestone.yaml:12]: Is `id` right here? Shouldn't this be `name`?

File: .github/workflows/auto-create-milestone.yaml [hunk 2]
```diff
@@ -8,14 +8,14 @@ jobs:
...
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           title: d-
           days: Friday
           count: 4
           format: YYYY.MM.DD
 
-      - name: result
+      - name: echo
```
Comment (erunion) [inline .github/workflows/auto-create-milestone.yaml]:
```suggestion
      - name: result
```

**Review by erunion — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-create-milestone.yaml (hunk 1 above)
Comment (seunggabi) [reply .github/workflows/auto-create-milestone.yaml:12]:
```
{{ steps.run.outputs.milestones }}
```

`id` is works.
`name` is works?

Comment (erunion) [reply .github/workflows/auto-create-milestone.yaml:12]: I don't follow. Why the change from `name` to `id`?

Comment (seunggabi) [reply .github/workflows/auto-create-milestone.yaml:12]: @erunion 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-output-as-url

please read this.
name don't work

id is correct

**Review by erunion — APPROVED** [review]

**Pull request merged**